sources/packages/backend/apps/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts

```diff
@@ -122,10 +119,7 @@ export class ProgramInfoRequestInstitutionsController extends BaseController {
     // assessment always available for submitted student applications).
     // PIR process happens only during original assessment.
     // Offering available only when PIR is completed.
-    const originalAssessmentOffering = application.studentAssessments.find(
-      (assessment) =>
-        assessment.triggerType === AssessmentTriggerType.OriginalAssessment,
-    ).offering;
+    const originalAssessmentOffering = application.currentAssessment.offering;
```

Comment:
I am not following why this one was changed. The PIR offering belongs to the "Original Assessment" only.
The current assessment may contain an offering from a reassessment that is no longer the PIR-provided offering. If I am missing something, please let me know.